integrations/extensions/docs/elasticsearch-install-and-setup/text_embedding_deploy_and_use.md

```diff
@@ -219,4 +223,9 @@ curl -X GET "${ES_URL}/${ES_EMBEDDING_INDEX_NAME}/_search" \
     "text"
   ]
 }'
-```
\ No newline at end of file
+```
+
+## What's next?
+This guide is part of starter kits for setting up a custom Elasticsearch extension for conversational search in IBM watsonx Assistant. You can refer to the guides below for more context:
```

Comment:
I really like that you added a What's next?  section.  I suggest making it a little more specific than just copying what you had at the top.   What do you think is the likely thing they should do next if they are reading this as part of another guide flow?  Even listing two possible alternatives things they would do next may help

Reply:
Good point. I've added more clear next steps.